### DISCUSSION TITLE
add tw- class prefixes at build time

### DISCUSSION THREAD

**Discussion by techopswhiz** [tailwindlabs/tailwindcss#9708] [Ideas]
I need to use prefixes in the build output, but I don't want to degrade the developer experience. Is there a way to add the prefix only at build time?

**Comment by Amyconnell** [comment]
Use display: flex to create a container element. Remove the border and outline from the <input> field. Use the :focus-within pseudo-class selector to style the parent element accordingly, when the user interacts with the <input> field.   
https://www.myccpay.me

**Reply by techopswhiz** [reply]
is this spam?
